omod/src/main/java/org/openmrs/module/sync2/web/controller/Sync2ModuleController.java

```diff
@@ -94,17 +100,16 @@ public String manualPush(ModelMap model) {
 	 */
 	@RequestMapping(value = "/manualPull")
 	public String manualPull(ModelMap model) {
-		try {
-			LOGGER.info("Start Parent Feed Reader...");
-			ParentFeedReader parentFeedReader = ContextUtils.getParentFeedReader();
-			parentFeedReader.pullAndProcessAllFeeds();
+		
+		LOGGER.info("Start Parent Feed Reader...");
+		ParentFeedReader parentFeedReader = ContextUtils.getParentFeedReader();
+		parentFeedReader.pullAndProcessAllFeeds();		
+		Integer lastAuditMsgId = syncAuditService.getCountOfMessages().intValue();
+		Boolean succes = syncAuditService.getMessageById(lastAuditMsgId).getSuccess();
```

Comment:
I don't know, but did you intend to use `Boolean success `rather than `Boolean succes`